### ISSUE TITLE
Type lowecase character in Apple 2 emulator

### ISSUE THREAD

**Issue by wyatt-wong** [whscullin/apple2js#148]
How do I type lowecase character in Apple 2 emulator ?

**Comment by whscullin** [comment]
It's not easy to get an Apple ][ or ][+ to do lower case. There are a handful of programs that support lower case, mostly word processors. If you want lower case you're better off with the Apple //e emulator.

**Comment by Bischoff** [comment]
One way is to switch to 80 columns mode (PR#3 from BASIC if I remember well).

**Comment by wyatt-wong** [comment]
That's what I saw from the Option screen:

![image](https://user-images.githubusercontent.com/10095648/185519887-cc5e1baa-7b6f-491b-926f-1f65b66b14e5.png)

**Comment by whscullin** [comment]
Yeah, I haven't been able to find a good piece of software to showcase for that. The built in ROM didn't change, so only 3rd party software made use of it. Oo-Topos will use it, and possibly some other interactive fiction titles. Apple Writer may also use it.

If you do PR#3 and enter 80 column mode, you can switch between lowercase and uppercase with ctrl-A. With the lowercase ROM, if you reset out of 80 column mode, you can still see upper and lower case.
